### DISCUSSION TITLE
A simple summary of the mod update progress for those who are curious / 模組更新進度短報

### DISCUSSION THREAD

**Discussion by ganha22017** [inmny/Cultivation-Way-Core#6] [General]
Eng:

I kept opening and closing this page countless times for months because I can't wait for the next update, then I realised there might be curious people like me, so I decided to write this essay for you to get a quick view of "what's going on", without having to constantly check between different websites. 

Just like me, you're curious enough to click the GitHub link, trying to dig some progress at least.
The last update on Steam was on 24/12/2023, you might be thinking "The modders must be too busy IRL, they had to abandon this mod, and maybe half a year later they will come back."

Well, if you check Inmny's main page —— I believe Inmny is "one of the" main contributors (This mod has some story behind it, CWCore was supposed to be maintained by a group called 启源工作室, yet these people are not mentioned on the Steam workshop, only two Steam account on the author list)—— Inmny is active on another project, such as [this mod](https://github.com/inmny/CW_FanstasyCreature), which is an extension module of CWCore, it is confirmed that developers didn't drop the mod, or in the worst case, only Inmny and Moouqi are still working on it. It seems that they're just ……slow on the project, but they will cook eventually.

Hope your curiosity is relieved.
29/4/2024

中文：

最近幾個月，我反復點進來GitHub查進度，我實在想知道下次更新什麽時候到。之後我想到，可能某天會有其他好奇的人點進來看進度如何，所以就寫了點東西試著「總結現況」，如此一來你就不用為了綫索而費力找來找去。

總之你和我一樣，好奇得發慌，就順著那條鏈接進來GitHub，看看能不能找到些蛛絲馬跡。
Steam上最近一次更新已經是24/12/2023的事了，你可能會覺得：「應該是大佬們現實裏都有事要忙吧，只能斷更了，也許半年後他們就忙完了呢」

事實上呢，如果你有看過Inmny的個人主頁——我猜Inmny是mod的主要開發者之一（修仙mod的開發好像是幾經轉手，目前由启源工作室負責維護，但Steam上的建立者名單也沒看到那些成員的賬號）——Inmny沒有潛水，他最近有在開發[這個mod](https://github.com/inmny/CW_FanstasyCreature)，正巧這個mod是啓源修仙的拓展模組，基本可以認為mod的開發仍在進行中，最壞情況下，可能只剩Inmny 和 Moouqi仍然在寫mod，但也沒有中斷開發一說。 他們大概只是……碼字比較慢而已，只能等了。

希望有解答到你。
29/4/2024

**Comment by daybreak1929** [comment]
其实启源工作室的修仙一直都是由一米(Inmny)负责，仅在0.13~0.14期间因一米本人忙于升大学所以交给其他人，0.14后期的1.0.0就是一米开发的，如今的情况和当初相同，但一米开发的东西多了很多，比如[开源模组装载器NeoModLoader](https://github.com/WorldBoxOpenMods/ModLoader)，协助开发[春秋/古代战争2.0](https://github.com/goatdie/AncientWarfare2.0)，所以进度相比之前慢了很多，估计只能等国内暑假了
启源工作室中Inmny是主要代码，Moouqi是主要画师，剩下的不是策划就是去搞了其他东西而已
本人不是一员，只是待得久确实知道一些，国内模组区主要还是一米等大佬在负重前行:(
